azure-pipelines.yml

```diff
@@ -82,10 +86,19 @@ jobs:
         displayName: Add relavent channels
 
       - script: |
-          conda create -n test-environment python=$(python.version) --file requirements-dev.txt
+          conda update -c defaults conda
+          conda update --all
+          conda create -n test-environment
           call activate test-environment
-          pip install -e .
-        displayName: Create Conda env, Activate, Install dependencies, Install Branch
+          conda install --yes python=$(python.version) --file requirements-dev.txt
+          conda install --yes python=$(python.version) --file requirements-dev.txt
+        displayName: Create Conda env, Activate, Install dependencies
+        
+      - script: |
+          call activate test-environment
+          python -m pip --version
+          python -m pip --verbose install -e .
+        displayName: Install mBuild
```

Comment:
Should this change (doing `pip install -e .` in a separate step) be mirrored in foyer?

Reply:
Yeah. We should probably mirror all of these changes to the `azure-pipelines.yml` in foyer since we were getting the HTTP errors there too.